src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocument.cs

```diff
@@ -30,9 +30,10 @@ public class BuildTocDocument : BuildTocDocumentStepBase, ISupportIncrementalBui
         /// </summary>
         public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models, IHostService host)
         {
-            var resolvedModels = TocHelper.Resolve(models, host).ToList();
+            var referencedTocModels = new Dictionary<string, FileModel>();
```

Comment:
This can be a `HashSet`. It can be an output of `TocHelper.Resolve` instead of input. Use `OSPlatformSentiveComparer` to construct it.

Reply:
Thanks @superyyrrzz , I've updated code accordingly. Please help review again